container-images/assign-class-label/mutate.py

```diff
@@ -0,0 +1,180 @@
+import logging
+import json
+import base64
+from flask import Flask, request, jsonify, Response
+from kubernetes import config, client
+from openshift.dynamic import DynamicClient
+
+from pydantic import BaseModel, ValidationError, constr
+from typing import Dict, Any, Optional, List
+
+LOG = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+class PodMetadata(BaseModel):
+    labels: Optional[Dict[str, str]] = None
+
+
+class PodObject(BaseModel):
+    metadata: PodMetadata
+
+
+class AdmissionRequest(BaseModel):
+    uid: constr(min_length=1)
+    object: PodObject
+
+
+class AdmissionReview(BaseModel):
+    request: AdmissionRequest
+
+
+def decode_pod_user(pod_user: str) -> str:
+    user = pod_user.replace('-40', '@').replace('-2e', '.')
+    return user
```

Comment:
Since you're not using the return value inside this function, you can drop the local variable:

```suggestion
    return pod_user.replace('-40', '@').replace('-2e', '.')
```